fuel-vm/src/interpreter/contract/tests.rs

```diff
@@ -53,19 +55,49 @@ fn test_contract_balance(b: Word, c: Word) -> Result<(), RuntimeError> {
 }
 
 #[test_case(true, 0, 50, 32 => Ok(()); "Can transfer from external balance")]
-fn test_transfer(external: bool, a: Word, b: Word, c: Word) -> Result<(), RuntimeError> {
+#[test_case(false, 0, 50, 32 => Ok(()); "Can transfer from internal balance")]
+#[test_case(true, 0, 70, 32 => Err(RuntimeError::Recoverable(PanicReason::NotEnoughBalance)); "Cannot transfer from external balance insufficient funds")]
+#[test_case(false, 0, 70, 32 => Err(RuntimeError::Recoverable(PanicReason::NotEnoughBalance)); "Cannot transfer from internal balance insufficient funds")]
+fn test_transfer(
+    external: bool,
+    contract_id_offset: Word,
+    transfer_amount: Word,
+    asset_id_offset: Word,
+) -> Result<(), RuntimeError> {
+    // Given
+
+    const ASSET_ID: [u8; AssetId::LEN] = [2u8; AssetId::LEN];
```

Comment:
You can create the type itself. The same is applicable to `ContractId` and `Adress`=) And instead of `&ASSET_ID[..]` you can use `ASSET_ID.as_ref()`

```suggestion
    const ASSET_ID: AssetId = AssetId::new([2u8; AssetId::LEN]);
```